### PR TITLE
[hail] spark 2.4.5, scala 2.12, dataproc 1.5

### PR DESCRIPTION
**Pull request by danking**

### CODE REVIEW

File: hail/src/main/scala/is/hail/io/tabix/TabixReader.scala [hunk 1]
```diff
@@ -117,7 +116,6 @@ class TabixReader(val filePath: String, fs: FS, idxFilePath: Option[String] = No
       fatal(s"Hail only supports tabix indexing for VCF, found format code ${ format }")
     val colSeq = readInt(is)
     val colBeg = readInt(is)
-    val colEnd = readInt(is)
```
Comment (tpoterba) [inline hail/src/main/scala/is/hail/io/tabix/TabixReader.scala:120]: this is a bug.

File: hail/src/main/scala/is/hail/linalg/BlockMatrix.scala [hunk 2]
```diff
@@ -1961,7 +1959,6 @@ class BlockMatrixReadRowBlockedRDD(
     val is = fs.open(filename)
     val in = BlockMatrix.bufferSpec.buildInputBuffer(is)
 
-    val rows = in.readInt()
```
Comment (tpoterba) [inline hail/src/main/scala/is/hail/linalg/BlockMatrix.scala:1964]: bug

File: hail/src/main/scala/is/hail/io/tabix/TabixReader.scala (hunk 1 above)
Comment (danking) [reply hail/src/main/scala/is/hail/io/tabix/TabixReader.scala:120]: paging @chrisvittal , can you either push a fix to this branch or explain to where I should thread colend?

Comment (danking) [reply hail/src/main/scala/is/hail/io/tabix/TabixReader.scala:120]: Or can I just read the int and drop it?

Comment (tpoterba) [reply hail/src/main/scala/is/hail/io/tabix/TabixReader.scala:120]: just don't bind it to a variable. readInt() is side-effecting in that it moves the cursor. The value isn't needed here

**Comment by tpoterba** [comment]
test module doesn't compile

**Comment by danking** [comment]
ugh there's more issues and build_hail isn't any faster. I'll have to look into this more later.

**Pull request closed without merging**